config/samples/kepler.system_v1alpha1_kepler.yaml

```diff
@@ -8,10 +8,15 @@ metadata:
     app.kubernetes.io/managed-by: kustomize
     app.kubernetes.io/created-by: kepler-operator
   name: kepler
-  # namespace: kepler
 spec:
   # TODO(user): Add fields here
   collector:
-    image: quay.io/sustainable_computing_io/kepler:release-0.4
+    image: quay.io/sustainable_computing_io/kepler:latest
```

Comment:
do we provide latest image as official release?

Reply:
I believe we do. @piparul can you confirm? This is just a sample of how the CRD would look like for the kepler operator so any image version theoretically can be passed to the operator when deploying kepler or kepler-model-server.

Reply:
Yes we do. Whenever we cut a release, we provide an official release image. https://quay.io/repository/sustainable_computing_io/kepler-operator?tab=tags